<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Szpital</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Użycie</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>256</width>
        <height>271</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Sprzęt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilość</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="amountUsedSpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>330</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-99</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="equipmentUsedLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>129</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addUsageButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Zapas</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>256</width>
        <height>271</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Sprzęt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilość</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
